org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "/app/fireflyluciferin/bin/FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=all",
+    "--filesystem=home",
+    "--env=LD_LIBRARY_PATH=/app/lib",
+    "--env=GTK_THEME=Yaru:dark",
+    "--env=FLATPAK_ENV=1"
+  ],
+  "modules": [
+    "shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json",
+    {
+      "name": "fireflyluciferin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x FireflyLuciferinLinux.deb",
+        "tar xvf data.tar.zst",
+        "mkdir -p /app/fireflyluciferin",
+        "echo -e '\nflatpak.env=true' >> opt/fireflyluciferin/lib/app/classes/project.properties",
+        "cp -r opt/fireflyluciferin/* /app/fireflyluciferin",
+        "install -Dm644 org.dpsoftware.FireflyLuciferin.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+        "install -Dm644 org.dpsoftware.FireflyLuciferin.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png",
+        "install -Dm644 org.dpsoftware.FireflyLuciferin.appdata.xml /app/share/share/appdata/${FLATPAK_ID}.appdata.xml"
+      ],
+      "cleanup": [
+        "/opt"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "org.dpsoftware.FireflyLuciferin.desktop"
+        },
+        {
+          "type": "file",
+          "path": "org.dpsoftware.FireflyLuciferin.appdata.xml"
+        },
+        {
+          "type": "file",
+          "path": "org.dpsoftware.FireflyLuciferin.png"
+        },
+        {
+          "type" : "file",
+          "url" : "https://dpsoftware.org/FireflyLuciferinLinux.deb",
+          "sha256" : "6c3421bd442d83f4d1eeaa1a3b0db5207ecb5e02cedb67f57f9961f8d16ebe97"
+        }
+      ]
+    },
+    {
+      "name": "cuda-toolkit",
+      "only-arches": [
+        "x86_64"
+      ],
+      "cleanup": [
+        "/cuda"
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir /app/cuda",
+        "sh cuda_toolkit.run --silent --toolkit --toolkitpath=/app/cuda",
+        "mv /app/cuda/lib64/libcudart.* /app/lib/",
+        "mv /app/cuda/lib64/libcublas* /app/lib/",
+        "mv /app/cuda/lib64/libnvrtc* /app/lib/"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "only-arches": [
+            "x86_64"
+          ],
+          "url": "https://developer.download.nvidia.com/compute/cuda/12.6.3/local_installers/cuda_12.6.3_560.35.05_linux.run",
+          "dest-filename": "cuda_toolkit.run",
+          "md5": "29d297908c72b810c9ceaa5177142abd"
+        }
+      ]
+    },
+    {
+      "name": "libnotify",
```

Comment:
libnotify is in the runtime.

Reply:
but java foreign function does not get it if I not bundle it. please leave this, this way.

Reply:
why not fixing the code? If you look for a `.so` you look for it wromng. soversion are a thing. Doing otherwise is broken by design.

Reply:
at worse you can make a symlink to the runtime provided version. This doesn't fix the terrible code but it works around it. There is definitely no need to build this here.

Reply:
@hfiguiere you are saying that Oracle is wrong with the latest Java Foreign Functions and Memory API?
it's a billionair project. that creates bindings over the .so files :)

ok in doing a symlink. do you know if freedesktop bundles libnotify because from the docs,
only GNOME runtime bundles it.
https://docs.flatpak.org/en/latest/available-runtimes.html

Reply:
@hfiguiere @bbhtt 
thank you a billion, I fixed it in my code without even the need for a sym link, 
sorry if I was hard understanding but I was not understood that.
fixed it now, thanks :)